Oje dev

AOjemann opened the pull request:
converted most keyboard controls to mouse controls, swap still on keyboard (planed for UI button)

AOjemann committed:
ani del, mouse fake

basic animation delegator implemented, hooked vomit and eat into master,
added some basic mouse touch fake controls

AOjemann committed:
fixed mouseEat (needs testing)

added mouse eating useing raycast, needs testing, if bugged look into
2dRay/2dCollider

Murkantilism commented:
Not a huge fan of the commented out code, if it's not being used delete it. Otherwise looks good.

AOjemann committed:
mobile touch controls, inventory open hook and mode swap

implemented touch version of controls
added inventory open controls and hook
put in compile comments to swap between mobile and computer controls

Murkantilism commented on GetUpChuckUnityProject/Assets/Scripts/MasterCtrl.cs:
```diff
@@ -5,46 +5,228 @@ public class MasterCtrl : MonoBehaviour {
 			swapPlayer();
 		}
+
+#if UNITY_EDITOR
```
This should be UNITY_STANDALONE || UNITY_EDITOR, so if we decide to publish for PC/OSX/Linux this region will compile.

Murkantilism commented on GetUpChuckUnityProject/Assets/Scripts/MasterCtrl.cs:
```diff
@@ -86,26 +281,41 @@ public class MasterCtrl : MonoBehaviour {
 	//walk right
 	void walkRight(){
 		activePlayer.moveRight();
+		//masterAnimationDel ("walkRight");
```
Why are some animation calls commented out while others aren't?

AOjemann committed:
removed dead comments, standalone compatibility
